### ISSUE TITLE
How to change placeholder name & icon??

### ISSUE THREAD

**Issue by dhanush-tydy** [weekwood/editorjs-video#2]
can we change placeholder name & icon

**Comment by SalahAdDin** [comment]
> can we change placeholder name & icon

What do you want to do?

**Comment by sebmeister2077** [comment]
> can we change placeholder name & icon

Yes @dhanush-tydy , you'll have to extend the class and implement your own `static get toolbox()` method

an example would be like this:
```js 
class MyVideoUploadTool extends VideoTool {
  static get toolbox() {
    return {
      icon: "<svg>....</svg>",
      title: "Title you want to display",
    };
  }
}
```

**Comment by SalahAdDin** [comment]
> > can we change the placeholder name & icon
> 
> Yes @dhanush-tydy, you'll have to extend the class and implement your own `static get toolbox()` method
> 
> an example would be like this:
> 
> ```js
> class MyVideoUploadTool extends VideoTool {
>   static get toolbox() {
>     return {
>       icon: "<svg>....</svg>",
>       title: "Title you want to display",
>     };
>   }
> }
> ```

Or, make a pull request to this repo, but it seems to be abandoned.
